pkgs/top-level/python-packages.nix

```
@@ -7202,6 +7206,8 @@ in {
 
   textfsm = callPackage ../development/python-modules/textfsm { };
 
+  textX = callPackage ../development/python-modules/textX { };
```

Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)
```suggestion
  textx = callPackage ../development/python-modules/textx { };
```